main.nf

```diff
@@ -493,13 +503,21 @@ if( params.skip_trimming ){
         tpc_r1 = params.three_prime_clip_r1 > 0 ? "--three_prime_clip_r1 ${params.three_prime_clip_r1}" : ''
         tpc_r2 = params.three_prime_clip_r2 > 0 ? "--three_prime_clip_r2 ${params.three_prime_clip_r2}" : ''
         rrbs = params.rrbs ? "--rrbs" : ''
+        multicore = ''
```

Comment:
Still not sure if this is the correct implementation 😕 See https://github.com/nf-core/atacseq/pull/65#issuecomment-558267887

Reply:
Addressed in https://github.com/nf-core/methylseq/pull/137/ 👍